docs/PACKAGING.md

```diff
@@ -0,0 +1,39 @@
+# Order of Operation
+
+## Command
+
+```bash
+npm run release
+```
+
+### Underneath the hood
+
+1. Create the `react` bundle inside the `build` directory, 
```

Comment:
Did you mean to do `1.` for all list items? Or a bullet point?

Reply:
Markdown automatically convert the 1 to the right count for list :D.